app/models/account.rb

```diff
@@ -90,6 +90,7 @@ def random_uid
 #  state                  :string(255)      default("pending"), not null
 #  created_at             :datetime         not null
 #  updated_at             :datetime         not null
+#  otp_enabled            :boolean          default(FALSE)
```

Comment:
don't place the column at the end of database table, last 2 should be 'created_at' and 'updated_at'